examples/main.go

```diff
@@ -34,13 +34,6 @@ func main() {
 
 	config := rpcvmx.NewConfig()
 
-	fmt.Println(config.SetString("foo", "bar"))
-	fmt.Println(config.String("foo", "foo"))
-
-	fmt.Println(config.SetInt("foo", 3))
-	fmt.Println(config.Int("foo", 0))
-
-	fmt.Println(config.SetBool("foo", false))
-	fmt.Println(config.Bool("foo", true))
+	fmt.Println(config.String("guestinfo.ovfEnv", ""))
```

Comment:
Probably want to undo this, and remove the binary examples/examples (add to .gitignore)

btw, maybe the example could be something like rpctool?  https://github.com/vmware/vic/blob/master/cmd/rpctool/main.go